lib/Util/Util.php

```diff
@@ -2,7 +2,52 @@
 
 namespace Stripe\Util;
 
+use Stripe\Account;
+use Stripe\AlipayAccount;
+use Stripe\ApplePayDomain;
+use Stripe\ApplicationFee;
+use Stripe\ApplicationFeeRefund;
+use Stripe\Balance;
+use Stripe\BalanceTransaction;
+use Stripe\BankAccount;
+use Stripe\BitcoinReceiver;
+use Stripe\BitcoinTransaction;
+use Stripe\Card;
+use Stripe\Charge;
+use Stripe\Collection;
+use Stripe\CountrySpec;
+use Stripe\Coupon;
+use Stripe\Customer;
+use Stripe\Dispute;
+use Stripe\EphemeralKey;
+use Stripe\Event;
+use Stripe\ExchangeRate;
+use Stripe\FileUpload;
+use Stripe\Invoice;
+use Stripe\InvoiceItem;
+use Stripe\InvoiceLineItem;
+use Stripe\IssuerFraudRecord;
+use Stripe\LoginLink;
+use Stripe\Order;
+use Stripe\OrderReturn;
+use Stripe\Payout;
+use Stripe\Plan;
+use Stripe\Product;
+use Stripe\Recipient;
+use Stripe\RecipientTransfer;
+use Stripe\Refund;
+use Stripe\SKU;
+use Stripe\Source;
+use Stripe\SourceTransaction;
 use Stripe\StripeObject;
+use Stripe\Subscription;
+use Stripe\SubscriptionItem;
+use Stripe\ThreeDSecure;
+use Stripe\Token;
+use Stripe\Topup;
+use Stripe\Transfer;
+use Stripe\TransferReversal;
+use Stripe\UsageRecord;
```

Comment:
@nickdnk Pretty minor, but given the size of this list, what do you think about just using the fully qualified path directly on the lines in the map below so we can avoid two of these big lists?

i.e., Something like this:

``` php
\Stripe\Account::OBJECT_NAME => 'Stripe\\Account',
```

Reply:
Hi @nickdnk! Thanks a lot for the contribution. Would you mind addressing @brandur-stripe's comment above? Once that's done, I think we'll be ready to merge :)